packages/@uppy/robodog/src/addProviders.js

```diff
@@ -3,6 +3,7 @@ const has = require('@uppy/utils/lib/hasProperty')
 
 const remoteProviders = {
   dropbox: require('@uppy/dropbox'),
+  box: require('@uppy/box'),
```

Comment:
excluding box from global module for now as it will initially be released as beta
```suggestion
```